mountpoint-s3/src/inode.rs

```diff
@@ -1635,6 +1647,42 @@ pub enum InodeError {
     },
 }
 
+impl InodeError {
+    /// Constructs InodeError::ClientError enriching metadata with error_code, bucket and key.
```

Comment:
Is `error_code` going to be anything other than `MOUNTPOINT_ERROR_CLIENT`?

Reply:
I can not think of a case for that now, will hardcode the code in the function body